#! /bin/sh /usr/share/dpatch/dpatch-run
## 11_allow-non-existing-dvd-drive.dpatch by Tobias Grimm <etobi@debian.org>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: Don't let VDR be unstartable, if the DVD drive is not accessible,
## DP: just log a warning and continue.

@DPATCH@
Index: vdr-plugin-dvd-0.3.6~b03+cvs20090426.0013/dvd.c
===================================================================
--- vdr-plugin-dvd-0.3.6~b03+cvs20090426.0013.orig/dvd.c	2011-04-09 17:23:43.000000000 +0200
+++ vdr-plugin-dvd-0.3.6~b03+cvs20090426.0013/dvd.c	2011-04-09 17:23:46.000000000 +0200
@@ -76,8 +76,7 @@
                 // fprintf(stderr, "arg: %s\n", optarg);
                 cDVD::SetDeviceName(optarg);
                 if (!cDVD::DriveExists()) {
-                    esyslog("vdr: DVD drive not found: %s", optarg);
-                    return false;
+                    esyslog("WARNING - DVD drive not found: %s\n", optarg);
                 }
                 break;
             default:
